lib/colorado_lottery.rb

```diff
@@ -0,0 +1,39 @@
+class ColoradoLottery
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants
+  def initialize(registered_contestants = {},
```

Comment:
you typically won't see parameters new-lined like this

Reply:
also, they don't need to be parameter defaults-- there are no instances of these values being passed in (and it probably makes sense that they should always start empty) so i think default values set inside `initialize` is a better choice here